In [5]:
import pandas as pd

df = pd.read_csv('data/heart.csv')
print(df.duplicated().sum())
print(df[df.duplicated()])

1
     age  sex  cp  trestbps  chol  fbs  restecg  thalach  exang  oldpeak  \
164   38    1   2       138   175    0        1      173      0      0.0   

     slope  ca  thal  target  
164      2   4     2       1  


### سوال 8
'''
آیا رکورد تکراری وجود دارد؟
'''

# تحلیل
'''
duplicated()
یه تابع در پانداس هست که سطر های تکراری رو از دیتافریم پیدا میکنه
و خروجی آن
False: یعنی این سطر تکراری نیست
True:یعنی این سطر تکراری هست


Sum()
در پانداس و پایتون جمع مقادیر رو حساب میکنه
دو تا کاربرد اصلی داره:
۱) وقتی روی لیست از عداد یا یه ستون عددی استفاده بشه مجموع اون عدادو برمیگردونه
۲) true :1
false:0
Sum
روی یه سری ۰ و ۱ تعداد ۱ هارو حساب میکنه

df[df.duplicated()
یه ماسک میسازه (سری true و false )

df.duplicated()
یه ماسک میسازه (سری true و false )

'''

# خروجی
'''
بله رکورد تکراری وجود دارد ولی فقط یک مورد است
تعداد سطر های تکراری = 1
نمایش خود اون سطر تکراری 164
'''